lib/assessment.md

```diff
@@ -138,8 +161,11 @@ div {
 ```
 
 <!-- your answer starts here -->
+This is my original statement which is wrong:
+div one would be floating to the right and divs two and three would be floating to the left.  Then divFour would start underneath cause it was cleared left. And both four and five would be next to each other.
 
-<!-- your answer ends here -->
+Ran code in codepen:
```

Comment:
YES! This is the deeper lesson that we want to you take away from this - test stuff out!